packages/pyright-internal/src/tests/fourslash/completions.importInterimFile.fourslash.ts

```diff
@@ -0,0 +1,52 @@
+/// <reference path="fourslash.ts" />
+
+// @filename: test1.py
+//// import al[|/*marker1*/|]
+
+// @filename: altair/__init__.py
+// @library: true
+//// """module docstring"""
+////
+//// __all__ = [ "selection_interval" ]
+////
+//// from .vegalite import (
+////     selection,
+////     selection_interval
+//// )
+
+// @filename: altair/vegalite/__init__.py
+// @library: true
+//// def selection(): pass
+//// def selection_interval(): pass
+
+// @filename: altair/py.typed
+// @library: true
+//// # has to contain something for file to be written
+
+{
+    // Force interim file to be created
+    // @ts-ignore
+    await helper.verifyCompletion('included', 'markdown', {
+        marker1: {
+            completions: [
+                { label: 'altair', kind: Consts.CompletionItemKind.Module, documentation: 'module docstring' },
+            ],
+        },
+    });
+
+    helper.replace(0, 9, 'import altair as alt\n\nalt.');
```

Comment:
what these 2 number `0` and `9` mean? what does `replace` actually replace?

Reply:
Changed the args to me more meaningful.